tests/e2e/tests/subscriptions/subscription-renewal.spec.js

```diff
@@ -0,0 +1,104 @@
+import { test, expect } from '@playwright/test';
+import config from 'config';
+import { payments, api } from '../../utils';
+
+const { setupCheckout, fillCardDetails } = payments;
+
+let productId;
+let username, userEmail;
+
+test.beforeAll( async () => {
+	// This allow multiple tests to run in parallel.
+	const randomString = Date.now();
+	userEmail = randomString + '+' + config.get( 'users.customer.email' );
+	username = randomString + '.' + config.get( 'users.customer.username' );
+
+	const user = {
+		...config.get( 'users.customer' ),
+		...config.get( 'addresses.customer' ),
+		email: userEmail,
+		username,
+	};
+
+	await api.create.customer( user );
+
+	const product = {
+		...config.get( 'products.subscription' ),
+		regular_price: '9.99',
+		meta_data: [
+			{
+				key: '_subscription_period',
+				value: 'month',
+			},
+			{
+				key: '_subscription_period_interval',
+				value: '1',
+			},
+		],
+	};
+
+	productId = await api.create.product( product );
+} );
+
+test.afterAll( async () => {
+	await api.deletePost.product( productId );
+} );
+
+test( 'customer can renew a subscription @smoke @subscriptions', async ( {
+	page,
+} ) => {
+	await test.step( 'customer login', async () => {
+		await page.goto( `/wp-admin` );
```

Comment:
Maybe we should go straight to `/wp-login.php` here instead of relying on a redirect from `/wp-admin`? I feel like maybe ensuring that we're actually on the login page (instead of being redirected to a wp-admin) might be a good idea 🤔 

Might even be a decent candidate for some sort of utility function?

Reply:
> Might even be a decent candidate for some sort of utility function?

Definitely. I've seen this error before. It usually works on a second retry. I'll move the login to an utility function.